src/community/blocks.tsx

```diff
@@ -0,0 +1,293 @@
+import * as React from 'react';
+import {sprinkles} from '../sprinkles.css';
+import Stack from '../stack';
+import * as styles from './blocks.css';
+import {Text2, Text3, Text5, Text8} from '../text';
+import {vars} from '../skins/skin-contract.css';
+import Inline from '../inline';
+import Box from '../box';
+import ProgressBar from '../progress-bar';
+import classNames from 'classnames';
+
+import type StackingGroup from '../stacking-group';
+import type Image from '../image';
+import type Tag from '../tag';
+import type {RendersNullableElement} from '../utils/renders-element';
+import type {ExclusifyUnion} from '../utils/utility-types';
+
+interface BlockContentProps {
+    title?: string;
+    description?: string | Array<string>;
+}
+
+const BlockContent: React.FC<BlockContentProps> = ({title, description}) => {
+    const normalizedDescription =
+        description && Array.isArray(description) ? (
+            description.map((paragraph, i) => (
+                <Text2 regular color={vars.colors.textSecondary} key={i}>
+                    {paragraph}
+                </Text2>
+            ))
+        ) : (
+            <Text2 regular color={vars.colors.textSecondary}>
+                {description}
+            </Text2>
+        );
+
+    return (
+        <div className={styles.column}>
+            {title && (
+                <Text3 regular color={vars.colors.textPrimary}>
+                    {title}
+                </Text3>
+            )}
+
+            {!!description && normalizedDescription}
+        </div>
+    );
+};
+
+interface RowBlockBaseProps {
+    title?: string;
+    'aria-label'?: string;
+}
+
+interface RowBlockWithDescription extends RowBlockBaseProps {
+    description?: string;
+}
+
+interface RowBlockWithStackingGroup extends RowBlockBaseProps {
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+}
+
+type RowBlockProps = ExclusifyUnion<RowBlockWithDescription & RowBlockWithStackingGroup>;
+
+export const RowBlock: React.FC<RowBlockProps> = ({
+    title,
+    stackingGroup,
+    description,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div
+            style={{
+                display: 'flex',
+                alignItems: 'center',
+            }}
+            aria-label={ariaLabel}
+        >
+            {(title || stackingGroup || description) && (
+                <Inline
+                    className={sprinkles({
+                        display: 'flex',
+                        flex: 1,
+                    })}
+                    space="between"
+                    alignItems="center"
+                >
+                    {title && (
+                        <Box paddingRight={32}>
+                            <Text2 regular>{title}</Text2>
+                        </Box>
+                    )}
+                    {stackingGroup ? (
+                        <>{stackingGroup}</>
```

Comment:
you dont need the fragment
```suggestion
                        stackingGroup
```